src/pages/api/users/stats.js

```diff
@@ -1,13 +1,18 @@
+import { getAllAttendanceObjects } from "../../../../server/actions/volunteerstats";
+
 const {
-  getEventsByUserID,
+  getAttendanceByUserID,
 } = require("../../../../server/actions/volunteerstats");
 
 export default async function handler(req, res, next) {
   if (req.method === "GET") {
-    let event = await getEventsByUserID(req.query.volunteer, next);
+    let attendances;
+    if (req.query.volunteer != "null")
+      attendances = await getAttendanceByUserID(req.query.volunteer, next);
+    else attendances = await getAllAttendanceObjects(next);
```

Comment:
Not sure if this was you or Jonathan, but we should just return an empty list if a userId isn't passed